PowerValidatedSolutions.psm1

```diff
@@ -32662,18 +32810,47 @@ Function Test-vROPSAuthentication {
 Export-ModuleMember -Function Test-vROPSAuthentication
 
 Function Test-vRLIConnection {
+    <#
+        .SYNOPSIS
+        Check network connectivity to a vRealize Log Insight cluster or node.
+
+        .DESCRIPTION
+        Checks the network connectivity to a vRealize Log Insight cluster or node.
+        Supports testing a connection on ports 443 (HTTPS), 80 (HTTP), 22 (SSH), 9000 (CFAPI), 9543 (CFAPI SSL), 514 (SYSLOG), 1514 (SYSLOG), 6514 (SYSLOG). Default: 443 (HTTPS).
+
+        .EXAMPLE
+        Test-vRLIConnection -server xint-vrops01.rainpole.io
+        This example checks network connectivity with a vRealize Log Insight cluster or node on default port, 443 (HTTPS).
+
+        .EXAMPLE
+        Test-vRLIConnection -server xint-vrops01.rainpole.io -port 443
```

Comment:
```suggestion
        Test-vRLIConnection -server xint-vrli01.rainpole.io -port 443
```

Reply:
Addressed in 14b2e73bdb02f296b8ecbcda1032260e254946ce.